packages/services/f-feature-management/test/getValue.test.js

```diff
@@ -0,0 +1,41 @@
+import featureManagement from '../src/index';
+
+const { getValue } = featureManagement('');
+
+describe.skip('Generic value retrieval', () => {
```

Comment:
Is there a plan for these to be unmarked as `.skip`?

Reply:
Yes, I'm somewhat doing TDD in the early stages. 

The e2e tests in particular will exercise almost the entire implementation once completed so they'll be un-skipped late in the process, although I may start posting the quantity that are passing on future PRs. 

I can start un-skipping the other tests a few PRs from now.